API/views.py

```diff
@@ -28,34 +31,50 @@ def get42(url, payload, token):
 	return response.json()
 
 def get_token(CODE):
-	url = 'https://api.intra.42.fr/oauth/token'
-	UID = os.environ.get("UID_APP")
-	SECRET = os.environ.get("SECRET_KEY_APP")
-
-	data = {
-		'grant_type': 'authorization_code',
-		'client_id':  UID,
-		'client_secret': SECRET,
-		'code': CODE,
-		'redirect_uri': 'http://localhost:8000/twofa'
-	}
-
-	response = requests.post(url, data=data)
-	return response.json()['access_token']
+	try:
+		url = 'https://api.intra.42.fr/oauth/token'
+		UID = os.environ.get("UID_APP")
+		SECRET = os.environ.get("SECRET_KEY_APP")
+
+		data = {
+			'grant_type': 'authorization_code',
+			'client_id':  UID,
+			'client_secret': SECRET,
+			'code': CODE,
+			'redirect_uri': 'http://localhost:8000/twofa'
+		}
+
+		response = requests.post(url, data=data)
+	
+		return response.json()['access_token']
+	except:
```

Comment:
if the case is "except" how does the user get notified? does he get a clear msg?